src/Appium.Net/Appium/Interfaces/IInteractsWithApps.cs

```diff
@@ -60,5 +60,12 @@ public interface IInteractsWithApps : IExecuteMethod
         /// Closes the current app.
         /// </summary>
         void CloseApp();
+
+        /// <summary>
+        /// Gets the State of the app.
+        /// </summary>
+        /// <param name="appId">a string containing the id of the app.</param>
+        /// <returns>an integer number representing the status. 0 is not installed. 1 is not running. 2 is running in background or suspended. 3 is running in background. 4 is running in foreground.</returns>
+        int GetAppState(string appId);
```

Comment:
would it be better to have an enumeration for the method result rather than simple int?

Reply:
```csharp
public enum AppState
{
    NotInstalled,
    NotRunning,
    RunningInBackgroundOrSuspended,
    RunningInBackgroud,
    RunningInForeground
}
```
Something like this?